package/src/components/KeyboardCompatibleView/KeyboardCompatibleView.tsx

```diff
@@ -21,6 +21,10 @@ import {
 
 import { KeyboardProvider } from '../../contexts/keyboardContext/KeyboardContext';
 
+type State = {
```

Comment:
Note to self: We should really consider moving this to a functional component whenever we have the bandwidth to do so. It is more convenient indeed to port changes from the RN upstream to our custom implementation whenever they're needed, but it's not a complicated component at all to keep it a class one.